primitives/keyring/src/dilithium2.rs

```diff
@@ -21,10 +21,7 @@ use std::{collections::HashMap, ops::Deref};
 
```

Comment:
What about `should_work()` test? Can we uncomment it now and it will pass? BTW. Please, give it more meaningful name ;)

Reply:
Adjusted, changed `should_work` => `should_sign_and_verify_correctly`